docs/deploying/deploy-smart-contracts.mdx

```diff
@@ -11,31 +11,45 @@ To deploy your smart contracts to a live network, there are a few things you nee
 
 ## 1. Configure your network
 
-Scaffold-ETH 2 comes with a selection of predefined networks. You can also add your custom network in:
```

Comment:
I think this line made sense? Maybe change it a bit: 

"To add your custom network: 

go to ... "

Reply:
I completely missed this, thank you